examples/bevy_ggrs/Cargo.toml

```
@@ -34,8 +34,8 @@ bevy = { version = "0.11", default-features = false, features = [
   # gh actions runners don't like wayland
   "x11",
 ] }
-ggrs = "0.9"
-bevy_ggrs = "0.13"
+ggrs = { version = "0.10" }
+bevy_ggrs = { version = "0.14" }
```

```suggestion
ggrs = "0.10"
bevy_ggrs = "0.14"
```

We use the shorthand elsewhere, when possible.

---

I've also applied this to a couple of other instances for consistency